index.Rmd

```diff
@@ -128,25 +148,33 @@ hist(titanic$SibSp, col = "blue", border = "black",
 ```{r}
 #Parch
 
-summary(titanic$Parch)
+summary(titanic_transform$Parch)
 ```
 
 #### Sex
 
 ```{r}
-summary(titanic$Sex)
-unique(titanic$Sex)
+```
+
+```{r}
+#Codificamos el sexo como 0=female, 2=male
+titanic_transform <- titanic_transform %>% mutate(Sex = replace(Sex, Sex == "female", 0))
```

Comment:
Hey Cristian, awesome work! Just for passing the tests, I would say If you put a line break after %>% It will work :crossed_fingers:

Reply:
Thank you for the feedback. I wil try it now!